content/applications/services/helpdesk/advanced/track_and_bill.rst

```diff
@@ -0,0 +1,255 @@
+This means an invoice can be generated and payment can be received for this product before any
+timesheets entries have been recorded for these services.
+
+.. image:: track_and_bill/track-bill-product-prepaid-fixed.png
+   :align: center
+   :alt: The Service on Timesheets product with emphasis on the invoicing policy field.
+
+Finally, set the :guilabel:`Sales Price`, and confirm that the :guilabel:`Unit of Measure` is set to
+:guilabel:`Hours`.
+
+Invoice prepaid support services
+================================
+
+When support services are billed on a fixed price, an invoice can be created before any work is
+completed on the issue. In this case, a service product with the *Invoicing Policy* set to
+*Prepaid/Fixed Price* would be used, just like the one created in :ref:`the section above
+<helpdesk/configure-service-products>`.
+
+Create a sales order with prepaid product
+-----------------------------------------
+
+To invoice a customer for prepaid support services, first create a sales order (SO) with the support
+services product. To do this, go to :menuselection:`Sales app --> Orders --> Quotations`. Then,
+click :guilabel:`Create` to reveal a blank quotation form.
+
+Then, fill out the quotation form with the customer information.
+
+Go to the :guilabel:`Order Lines` tab of the quotation and click :guilabel:`Add a Product`. Then,
+select the *prepaid services product* configured in the steps above. Update the :guilabel:`Quantity`
+field with the number of hours.
+
+After updating any other necessary information, :guilabel:`Confirm` the quotation. This converts the
+quotation into an :abbr:`SO (sales order)`.
+
+Create and send an invoice for prepaid services
+-----------------------------------------------
+
+Once the :abbr:`SO (sales order)` has been confirmed, click the :guilabel:`Create Invoice` button.
+This opens a :guilabel:`Create invoices` pop-up window.
+
+If no down payment is collected, the :guilabel:`Create Invoice` type can remain as
+:guilabel:`Regular Invoice`. If a :doc:`down payment <../../../sales/sales/invoicing/down_payment>`
+is collected, choose between either :guilabel:`Down payment (percentage)` or :guilabel:`Down payment
+(fixed amount)`.
+
+When the necessary information has been entered, click :guilabel:`Create and View Invoice` or
+:guilabel:`Create Invoice`.
+
+.. tip::
+   Invoices are created in draft mode, so they can be reviewed and edited if necessary.
+
+The invoice can then be :doc:`sent to the customer <../../../finance/accounting/customer_invoices>`
+for payment.
+
+Create helpdesk ticket for prepaid services
+-------------------------------------------
+
+To create a *Helpdesk* ticket for prepaid services, navigate to :menuselection:`Helpdesk` and click
+the :guilabel:`Tickets` button to reveal a specific team's pipeline. Click :guilabel:`Create` to
+create a new ticket.
+
+On the blank ticket form, create a ticket :guilabel:`Title`, and enter the :guilabel:`Customer`
```

Comment:
```suggestion
On the blank ticket form, enter a ticket :guilabel:`Title` and the :guilabel:`Customer`
```
create just sounded odd since you type it in- a silly personal suggestion =)